kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -464,7 +485,7 @@ class SolvingStrategy
     ///@name Member Variables
     ///@{
 
-    ModelPart& mrModelPart;
+    ModelPart* mpModelPart = nullptr;
```

Comment:
I recently read about [`std::ref`](https://en.cppreference.com/w/cpp/utility/functional/ref), I think this could have also be used here

Reply:
The problem is that I need to be able to define an empty constructor, and therefore it is not possible to define a reference

Reply:
good point
I was thinking that std::ref can be reassigned, but not that is cannot be empty